index.js

```diff
@@ -141,8 +145,8 @@ export default class Delaunator {
             const x = coords[2 * i];
             const y = coords[2 * i + 1];
 
-            // skip duplicate points
-            if (x === xp && y === yp) continue;
+            // skip near-duplicate points
+            if (k > 0 && Math.abs(x - xp) <= EPSILON && Math.abs(y - yp) <= EPSILON) continue;
```

Comment:
:broken_heart: 
Delaunator was so good in triangulating while keeping the near-duplicates.

Reply:
The epsilon is 2^-52, so I think it's fine :)